src/cli/install.rs

```diff
@@ -261,13 +261,22 @@ fn print_post_install_message(settings: &Settings,
         }
     }
     match init_result {
-        Ok(true) => {
+        Ok(InitResult::Initialized) => {
             print_markdown!("\n\
                 `edgedb` without parameters will automatically\n\
                 connect to the initialized project.\n\
             ");
         }
-        Ok(false) => {
+        Ok(InitResult::Refused | InitResult::NonInteractive) => {
+            print_markdown!("\n\
+                To install the EdgeDB server and initialize the project, run\n\
+                the following:\n\
```

Comment:
```suggestion
                To install an EdgeDB version and initialize the project, run\n\
                the following command:\n\
```

@colinhacks I guess this is the new language in light of https://github.com/edgedb/edgedb-cli/pull/428?

Reply:
In the end I shortened this to this:

```
To initialize a new project, run `edgedb project init`
```